content/en/docs/_index.md

```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
 ### Declarative UI controls
 Operators can behave like managed service providers. Their user interface on the command line are APIs. For graphical consoles OLM annotates those APIs with descriptors that drive the creation of rich interfaces and forms for users to interact with the Operator in a natural, cloud-like way. 
 
+
+## How OLM solves the cluster admin and developer needs.
+
+### Operator Installation and Management
+
+This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills necessary metadata for visualizing them in compatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admin, developers to manage the lifecycle of all Operators and their associated services running across their clusters.
+
+### Dependency Resolution
+
+Using OLM eliminates the need for developers and cluster admins to have to worry about managing dependancies when developing one or more operators. OLM offers dependency resolution and upgrades the lifecycle of running operators. This functionality helps operator developers to resolve the problems related to dependency management.
+
+### Works similar to package Managers
+
+Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. If they are not available, then OLM will not install the operator.
```

Comment:
Please remove comma so that it reads as follows:
Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. If they are not available then OLM will not install the operator.

Reply:
Modified in #135